Cargo.toml

```diff
@@ -15,6 +15,7 @@ version = "0.6.0"
 [dependencies]
 byteorder = "0.5.3"
 leb128 = "0.2.1"
+getopts = "0.2"
```

Comment:
This should be under `[dev-dependencies]`, which are only included for tests and examples.

http://doc.crates.io/specifying-dependencies.html#development-dependencies